docs/querying/api.md

```diff
@@ -675,9 +675,9 @@ GET /api/v1/rules
 URL query parameters:
 
 - `type=alert|record`: return only the alerting rules (e.g. `type=alert`) or the recording rules (e.g. `type=record`). When the parameter is absent or empty, no filtering is done.
-- `rule_name[]=<rule name>`: return the groups and rules of the specified alerting and recording rules names, the parameter supports repetition. If we've filtered out all the rules of a group, the group is not returned. When the parameter is absent or empty, no filtering is done.
-- `rule_group[]=<group name>`: return the group and rules of the specified rule group names, the parameter supports repetitions. When the parameter is absent or empty, no filtering is done.
-- `file[]=<filepath>`: return the group and rules of specified filepath for rule groups, the parameter supports repetition. When the parameter is absent or empty, no filtering is done.
+- `rule_name[]=<rule name>`: return the groups and its rules of the specified alerting and recording rules names, the parameter supports repetition. If we've filtered out all the rules of a group, the group is not returned. When the parameter is absent or empty, no filtering is done.
```

Comment:
For the placeholder I would just use `<string>` everyhwere, because otherwise we usually define it at the top of the page as a new placeholder type.

Hmm, and I think not all of the plural / singular grammar matches up in the description ("groups and its" and "rules names")? How about something simpler like:

```suggestion
- `rule_name[]=<string>`: only return rules with the given rule name. If the parameter is repeated, rules with any of the provided names are returned. If we've filtered out all the rules of a group, the group is not returned. When the parameter is absent or empty, no filtering is done.
```

And similar for the others.

Reply:
I have done a bit more wordsmithing:

- removed the filtering comment from files and groups - it only applies to rules.
- included your suggested wording for the first sentence
- included the entity name in the second sentence

My only nit is that the first sentence is not entirely accurate as suggested - what we return in all cases is rule groups (because it's what the API returns) so saying something like "only return rules" is not entirely accurate, but perhaps this is OK? As in reality, what people come to look at here is rules.